webapp/app/session/route.js

```diff
@@ -16,7 +16,7 @@ export default Ember.Route.extend(
 
     model({ id }) {
       let self = this;
-      return self.store.findRecord("session", id).then(function(session) {
+      return self.store.queryRecord("session", { filter: { logical_id: id }} ).then(function(session) {
```

Comment:
@maormarcus this is not good, as some cases might not have logical_id assigned at all...